openedx/adg/lms/webinars/validators.py

```diff
@@ -0,0 +1,30 @@
+"""
+Validators for webinars
+"""
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.utils.translation import gettext_lazy as _
+
+from .helpers import extract_emails_from_string
+
+
+def validate_comma_separated_emails(emails_string):
+    """
+    Validate a string comma seperated emails by checking if all the emails are valid.
+    Raise a validation error if any of the emails is invalid.
+
+    Args:
+        emails_string (str): String containing the comma separated emails
+    """
+    try:
```

Comment:
Instead of placing the whole block inside `try` can we actually use exception handling where we think exception will be raised?